### ISSUE TITLE
Question about NSP loss of MobileBERT pretraining

### ISSUE THREAD

**Issue by RookieZB** [google-research/google-research#501]
Hello, firstly thanks for sharing code and model. I notice "classifier_activation" is set to false in model config, which means the output logit of NSP won't be normalized using "tanh" and value will be quite large, like [3e6, 3e6]. Since I think MLM logit is normalized through the final layer normalization layer and won't be very large, is it correct to combine the small MLM loss with a huge NSP loss? or I miss some processing steps in the code?
Hope you can explain that for me, and please tell me the final loss of MLM and NSP. Thank you!
